components/hdw-nvs/nvs_manager.c

```diff
@@ -301,3 +301,59 @@ bool readNvsBlob(const char* key, void* out_value, size_t* length)
         }
     }
 }
+
+/**
+ * @brief Delete the value with the given key from NVS
+ *
+ * @param key The NVS key to be deleted
+ * @return true if the value was deleted, false if it was not
+ */
+bool eraseNvsKey(const char* key)
```

Comment:
This needs to get mocked in `emu_storage.c` too.

Reply:
Added! And I was able to at least compile the code in `paint-untested` so I merged that into here.